ucp/core.py

```diff
@@ -874,6 +883,10 @@ def get_config():
         return _get_ctx().get_config()
 
 
+def register_am_allocator(allocator, allocator_type):
```

Comment:
Would be good with a doc string here

Reply:
Good catch, added one in https://github.com/rapidsai/ucx-py/pull/707/commits/dcf0a8e7912b535791badb5dc594205ff9b97855 .